rosidl_typesupport_introspection_c/rosidl_typesupport_introspection_c-extras.cmake.in

```diff
@@ -2,6 +2,9 @@
 # rosidl_typesupport_introspection_c/
 #   rosidl_typesupport_introspection_c-extras.cmake.in
 
+# use the same type of library
+set(rosidl_typesupport_introspection_c_LIBRARY_TYPE "@rosidl_typesupport_introspection_c_LIBRARY_TYPE@")
```

Comment:
linter fixup in f2b2602a34191fe5061a32e5819407ce13c4fcb8 (example unstable job: [![Build Status](https://ci.ros2.org/buildStatus/icon?job=nightly_linux-aarch64_release&build=360)](https://ci.ros2.org/view/nightly/job/nightly_linux-aarch64_release/360/))